css/home-not-logged-in.styl

```diff
@@ -354,8 +354,11 @@ $flex-section
 
             h3, p
               a
-                text-decoration: none
                 color: inherit
+                display: block
```

Comment:
Are these styles only for the logged in state? If so, then these shouldn't be here since they'll also be applied when `home-page-not-logged-in` class is used in a parent element. I'd prefer these to be in the stylus file for the component they're used in. Maybe the `index.styl` in the `home-page-for-user` folder? Use BEM style naming there to be consistent, too, please.